sonar-plugin-api/src/main/java/org/sonar/api/utils/Durations.java

```diff
@@ -62,76 +60,84 @@ public Duration create(long minutes) {
   /**
    * Convert the text to a Duration
    * <br>
-   * Example : decode("9d 10 h") -&gt; Duration.encode("10d2h") (if sonar.technicalDebt.hoursInDay property is set to 8)
+   * Example : decode("9d 10 h") -&gt; Duration.encode("10d2h")
    * <br>
    * @throws IllegalArgumentException
    */
   public Duration decode(String duration) {
-    return Duration.decode(duration, hoursInDay());
+    return Duration.decode(duration, HOURS_IN_DAY);
   }
 
   /**
    * Return the string value of the Duration.
    * <br>
-   * Example : encode(Duration.encode("9d 10h")) -&gt; "10d2h" (if sonar.technicalDebt.hoursInDay property is set to 8)
+   * Example : encode(Duration.encode("9d 10h")) -&gt; "10d2h"
    */
   public String encode(Duration duration) {
-    return duration.encode(hoursInDay());
+    return duration.encode(HOURS_IN_DAY);
   }
 
   /**
    * Return the formatted work duration.
-   * <br>
-   * Example : format(Locale.FRENCH, Duration.encode("9d 10h"), DurationFormat.SHORT) -&gt; 10j 2h (if sonar.technicalDebt.hoursInDay property is set to 8)
    *
+   * @deprecated since 6.3 as the {@link Locale#ENGLISH} is always used. Use {@link #format(Duration)} instead
    */
+  @Deprecated
   public String format(Locale locale, Duration duration, DurationFormat format) {
-    return format(locale, duration);
+    return format(duration);
   }
 
   /**
    * Return the formatted work duration.
    * <br>
-   * Example : format(Locale.FRENCH, Duration.encode("9d 10h"), DurationFormat.SHORT) -&gt; 10j 2h (if sonar.technicalDebt.hoursInDay property is set to 8)
+   * Example : format(Locale.FRENCH, Duration.encode("9d 10h"), DurationFormat.SHORT) -&gt; 10d 2d
    *
+   * @deprecated since 6.3 as the {@link Locale#ENGLISH} is always used. Use {@link #format(Duration)} instead
    */
+  @Deprecated
   public String format(Locale locale, Duration duration) {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)